week-3/C-array-every/exercise.js

```diff
@@ -6,7 +6,9 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 
 // -- Complete this function -->
-function hasOnlyStudents() {}
+function hasOnlyStudents(name) {
+  return name >= 3;
```

Comment:
This function is called with a parameter which is an array of strings (the variable `group`). You can't compare that to a number (well, you can in Javascript but it will always be `false` :slightly_smiling_face:  ). You need to check that every element of the array parameter is included in the students array. This exercise is about the array method `every`, so you would expect to use it somewhere. Have anther look at the Readme file if you are stuck.